app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -212,4 +222,17 @@ describe('Week', () => {
       />,
     );
   });
+
+  test('should redirect back to entry page after successfully selecting a team', async () => {
+    render(
+      <Week entry={entry} league={league} NFLTeams={NFLTeams} week={week} />,
+    );
+
+    const selectedTeam = await screen.findAllByTestId('team-radio');
+    fireEvent.click(selectedTeam[0]);
```

Comment:
Why is selectedTeam and array? You should only be able to select 1 team.

Reply:
its just to simulate that the first one was clicked. selectedTeam is every team and it has to know that only the first one is being clicked for testing

Reply:
I think Shashi might be pointing out that the variable name is misleading.

`selectedTeam` implies the DOM element found only represents the selected team, but that isn't true. At the time you find it, no teams are selected ,and the element you have is actually "all the radio elements".

So it'd be much more accurate to call it `teamRadios`, and then `fireEvent.click(teamRadios[0])` accurately represents what you are doing in the test, which is clicking the first radio.